### ISSUE TITLE
pv creation fails with PVC.spec.resources.requests.storage is not a multiple of 512

### ISSUE THREAD

**Issue by apolovov** [aleofreddi/csi-sanlock-lvm#43]
Similar to https://github.com/aleofreddi/csi-sanlock-lvm/issues/34.

Creating pvc with storage request is non multiple of 512 causes error.
```
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: nonmul512
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
      storage: "11362347344"
  storageClassName: sharedlun
  volumeMode: Filesystem
```

```
  Warning  ProvisioningFailed    12s (x5 over 27s)  csi-sanlock-lvm.csi.vleo.net_csi-sanlock-lvm-provisioner-0_e5642b25-76ae-42dc-adfb-00f25f1e33c0  failed to provision volume with StorageClass "sharedlun": rpc error: code = Unknown desc = failed to create volume csl-v-oCR6aqE2yL_2b46egYSliYc2Jbc_5KgGKzSEAsL2Yew@sharedlun: rpc error: code = Unknown desc = unexpected error: rc=3 stdout="" stderr="  Size is not a multiple of 512. Try using 11362347008 or 11362347520.\n  Invalid argument for --size: 11362347344b\n  Error during parsing of command line.\n"
```

**Comment by aleofreddi** [comment]
Thanks for reporting this one - I'll take a look.

**Comment by aleofreddi** [comment]
Fixed in `2ed47333bf101564c8d2dc58414e157d7f7cbc6c`.
